core/src/main/scala/cats/Parallel.scala

```diff
@@ -312,6 +312,18 @@ object Parallel extends ParallelArityFunctions2 {
     P.sequential(ftab)
   }
 
+  /**
+   * Like `Foldable[A].foldMapA`, but uses the applicative instance
+   * corresponding to the Parallel instance instead
+   */
+  def parFoldMapA[T[_]: Foldable, M[_], A, B: Monoid](
+    ta: T[A]
+  )(f: A => M[B])(implicit P: Parallel[M]): M[B] = {
+    val fb: P.F[B] =
+      Foldable[T].foldMapA(ta)(a => P.parallel(f(a)))(P.applicative, implicitly)
```

Comment:
Would you mind using either `Monoid[B]` (or just `B` if you make the suggested change above) instead of `implicitly` here?

Reply:
Done.